index.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html>
+<head>
+   <link rel="stylesheet" type="text/css" href="styles.css">
+
+</head>
+<body>
+  <header id="header">
+    <img src="mdn-logo-sm.png" id="header-img" alt="A logo would make a good image here">
+    </br></br>
+<nav class="topnav" id="nav-bar">
+  <a class="nav-link" href="home.html">Home</a>
+  <a class="nav-link" href="buy.html">buy</a>
+  <a class="nav-link" id="video" href="watch.html">watch video</a>
+</nav>
+<video src="movie.mp4" id="video" width="320" height="240" controls>
+  Watch video
+</video>
+
+<h1 id="header">My Header</h1>
+
+<form action="https://www.freecodecamp.com/email-submit" id="form">
```

Comment:
You did a good job here with the action attribute, you should also use the method attribute as they both help screen readers.